website/content/docs/k8s/connect/ingress-gateways.mdx

```diff
@@ -62,80 +61,99 @@ as previously discussed.
 
 ## Configuring the gateway
 
-Now that Consul has been installed with ingress gateways enabled, you must add the corresponding configuration to Consul. This requires you to use the Consul CLI.
-Configuring the ingress gateway requires:
+Now that Consul has been installed with ingress gateways enabled, you must configure the gateways via the [`IngressGateway`](/docs/connect/config-entries/ingress-gateway) custom resource.
 
-- Accessing the Consul server
-- Submitting an Ingress Gateway configuration entry to Consul
+Here is an example `IngressGateway` resource:
 
-### Accessing the Consul server
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: IngressGateway
+metadata:
+  name: ingress-gateway
+spec:
+  listeners:
+    - port: 8080
+      protocol: http
+      services:
+        - name: static-server
+```
 
-You can access the Consul server directly from your host via `kubectl port-forward`, this is helpful for interacting with the Consul UI locally as well as validating connectivity of the application.
+Apply the `IngressGateway` resource with `kubectl apply`:
 
 ```shell-session
-$ kubectl port-forward consul-server-0 8500 &
+$ kubectl apply -f ingress-gateway.yaml
+ingressgateway.consul.hashicorp.com/ingress-gateway created
 ```
 
-If TLS is enabled use port 8501.
+Since we're using `protocol: http`, we also need to set the protocol of our service
```

Comment:
I'm not sure how this was working before without setting the protocol to http but it wasn't working for me without this.